.github/pull_request_template.md

```diff
@@ -0,0 +1,54 @@
+# Overview
+
+Addresses ticket [#TICKET-NUMBER](www.ticket-link.com)
+
+High level overview of changes; one or two sentences.
+
+The design for the ticket is based on:
+![This design from XXXXXX](screenshot-of-design.png)
+
+# Changes
+
+## Major
+
+- Major change one
+- Major change two
+- ...
+
+## Minor
+
+- Minor change one
+- Minor change two
+- Minor change three
+- ...
+
+## Non-functional changes (e.g. documentation)
+
+- Docs change one
+
+# Release requirements
+
+What steps are required for release, if any? E.g. Key Vault secrets, Terraform changes, etc.
+
+## Release requirement one title
+
+Release requirement one description
+
+- Step one
+- Step two
+- ...
+
+# Additional notes
+
+Any additional notes, comments, etc. that might be required here.
+
+# Checklist
+
+- [ ] Title uses [Angular commit convention](https://www.conventionalcommits.org/en/v1.0.0-beta.4/)
+- [ ] PR targets development branch
+- [ ] Unit tests have been added/updated
+- [ ] E2E tests have been added/updated
+- [ ] GitHub workflows/actions have been updated
+- [ ] Terraform has been updated
+- [ ] Documentation has been updated
+- [ ] Key Vault secrets have been added to dev
```

Comment:
```suggestion
- [ ] Any new Key Vault secrets have been added to dev
```
Trivial - but thinking minor tweaks like this and "Documentation has been updated where relevant" would makes me more inclined to look at it as a reviewer and think, is everything here clear? should we document? since we might not always change those, maybe more of a q style

Reply:
Good shout, amended.